src/components/chats/index.js

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from 'react';
+
+import { v4 as uuid } from 'uuid';
+
+import useTranslation from 'hooks/useTranslation';
+import { useGetConversationQuery } from 'services/conversation/conversation';
+
+import profile from 'assets/profile.svg';
+
+import './styles.scss';
+
+const Chats = () => {
+  const [matchList, setMatchList] = useState([]);
+  const [tabSelected, setTabSelected] = useState('HOME');
+
+  const { data: matchConversations } = useGetConversationQuery();
+  const user = JSON.parse(localStorage.getItem('user'));
+  const t = useTranslation();
+  const getMatches = () => {
+    const matches = matchConversations?.matches || [];
+    setMatchList(matches);
+  };
+
+  useEffect(() => {
+    getMatches();
+  }, [matchConversations]);
+
+  const handleClickOpenChat = () => {
+    setTabSelected('CHAT');
```

Comment:
What do you think about defining each tab name as a constant? Same for `HOME` in line 14. As a tip: if using TypeScript this could be a good fit for an Enum